source/agora/consensus/protocol/Nominator.d

```diff
@@ -1160,9 +1152,9 @@ extern(D):
             else if (this.consensus_data.tx_set.length < other.consensus_data.tx_set.length)
                 return 1;
 
-            if (this.hash < other.hash)
+            if (this.hash > other.hash)
```

Comment:
Is this a bug fix?

Reply:
Yes and no, It doesn't matter which order as long as it is deterministic. However it is easier for the test if the front is a non zero hash value so that adding any non Hash.init value will enable the last test case to work. Not sure why the test didn't have a hash value before.